### DISCUSSION TITLE
Download Link

### DISCUSSION THREAD

**Discussion by WimpyKidFan2007** [Techokami/SonicWorldsNext#120] [Q&A]
Where can I find the download link for Sonic Worlds Next?

**Comment by DimensionWarped** [comment]
This is a github repo. You download it from the main page of the repo. Usually you'd use a tool like either the git command line or the Github desktop to do the download for you since then you'll have access to the version control that git provides. If you are green and don't want to deal with any of that, you can download the source as a zip file from the main page too.

![image](https://github.com/Techokami/SonicWorldsNext/assets/7477617/0c2cf8f8-c1e3-458e-ad2e-aaf75012a7ca)

**Comment by UndertaleFan6** [comment]
the only proper option is to download the master branch either through git or downloading it as a zip
